classes/injector.php

```diff
@@ -26,6 +26,8 @@
 
 namespace tool_webanalytics;
 
+use watool_matomo\client;
```

Comment:
I don't think by design we should use anything from tool functionality directly as tools are submodules so they are pluggble. If you have to do this it's a sign that you are doing something wrong from the design point of view.